app/users/edit/credits/index/template.hbs

```diff
@@ -83,51 +112,72 @@
           <LoadingIcon />
         </div>
       {{else}}
-        {{#with this.overtimeCredits.lastSuccessful.value as |overtimeCredits|}}
-          <div class="card-block">
-            <div class="grid">
-              <div class="grid-cell">
-                <table class="table table--striped">
-                  <thead>
-                    <tr>
-                      <th>Valid as of</th>
-                      <th>Duration</th>
-                      <th>Comment</th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    {{#each overtimeCredits as |overtimeCredit|}}
-                      <tr class="{{if (can 'edit overtime-credit') 'pointer'}}" {{action (perform this.editOvertimeCredit) overtimeCredit.id on='click'}}>
-                        <td>{{moment-format overtimeCredit.date 'DD.MM.YYYY'}}</td>
-                        <td>{{humanize-duration overtimeCredit.duration false}}</td>
-                        <td>{{overtimeCredit.comment}}</td>
+        {{#let this.overtimeCredits.lastSuccessful.value as |overtimeCredits|}}
+          {{#if overtimeCredits}}
+            <div class="card-block">
+              <div class="grid">
+                <div class="grid-cell">
+                  <table class="table table--striped">
+                    <thead>
+                      <tr>
+                        <th>Valid as of</th>
+                        <th>Duration</th>
+                        <th>Comment</th>
                       </tr>
-                    {{/each}}
-                  </tbody>
-                </table>
+                    </thead>
+                    <tbody>
+                      {{#each overtimeCredits as |overtimeCredit|}}
+                        <tr
+                          class="{{if (can 'edit overtime-credit') 'pointer'}}"
```

Comment:
This will probably also fail aria checks from ember-template-lint?

Reply:
I meant the line underneath.

Reply:
sure it does :)